deployment/overlays/prune/master-clusterrole.yaml

```diff
@@ -1,3 +1,5 @@
-- op: add
-  path: "/rules/0/verbs/-"
-  value: "list"
+- op: remove
+  path: "/rules/0/verbs/3"
+
```

Comment:
We need to be able to list (but we don't need to have it in the rule two times

```suggestion
```

Simplest thing would be to just drop this patch (prune/master-clusterrole.yaml) altogether. It would give access to NFD CRDs but that doesn't really do any harm. But your call